lib/uploadcare/api/file_storage_api.rb

```diff
@@ -0,0 +1,29 @@
+module Uploadcare
+  module FileStorageApi
+    BATCH_SIZE = 100
+
+    def store_files(objects)
+      in_batches(objects) { |batch| put "/files/storage/", to_uuids(batch) }
+    end
+
+    def delete_files(objects)
+      in_batches(objects) { |batch| delete "/files/storage/", to_uuids(batch) }
+    end
+
+    private
+
+    def in_batches(enum)
+      enum.each_slice(BATCH_SIZE) { |batch| yield batch }
+    end
+
+    def to_uuids(objects)
+      objects.map do |object|
+        case object
+        when Uploadcare::Api::File then object.uuid
+        when /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/ then object
+        else raise(ArgumentError, "Unable to convert object to uuid: #{object}")
```

Comment:
@dmitry-mukhin I don't like the warning idea because it can be easily missed. Something is clearlly wrong with the code that tries to send us invalid values and an exception should be raised. 

But we can raise the exception earlier, before the first request.

Both endpoints accept an enumerable with Files/UUIDs:

- If this enumerable is a FileList, invalid values won't occur.
- If it is an array of UUIDs or Files we can check it all for invalid values before the first request. 
- Potentially it could be some other kind of enumerable containing Files/UUIDs. If so, user is responsible for that, not we

What do you think?